### ISSUE TITLE
Updating property with type "ANY" only works every other time (2.1.0)

### ISSUE THREAD

**Issue by elexx** [orientechnologies/orientdb#4790]
When using properties with type "ANY", then all following `UPDATE`s on this property do either:
* Set the property to the given value, if the last value was null
* Set the property to null, if the last value was not null

A shortened copy&paste from my console:
```
CREATE DATABASE memory:testdb
CREATE CLASS Test EXTENDS V
CREATE PROPERTY Test.value ANY
CREATE VERTEX Test

UPDATE (SELECT FROM Test) SET value = false

    Updated record(s) '1' in 0.004000 sec(s).

SELECT FROM Test

    ----+-----+------+-----
    #   |@RID |@CLASS|value
    ----+-----+------+-----
    0   |#11:0|Test  |false       
    ----+-----+------+-----

UPDATE (SELECT FROM Test) SET value = false

    Updated record(s) '1' in 0.004000 sec(s).

SELECT FROM Test

    ----+-----+------+-----
    #   |@RID |@CLASS|value
    ----+-----+------+-----
    0   |#11:0|Test  |null       
    ----+-----+------+-----
```
Interestingly enough a `UPDATE ... RETURN` always shows the right value, like `Updated record(s) '[Test#11:0{value:false} v8]' in 0.005000 sec(s).`, even if the next `SELECT ...` will return a `null`.

I also tried to set an Integer because I suspected some parsing problems with booleans, but that did the same strange with of setting and erasing the value.

This happens in OrientDB version 2.1.0 (Community Edition).

**Comment by elexx** [comment]
any news on this?

**Comment by luigidellaquila** [comment]
Hi @elexx 

I reproduced the problem, I'll work on this in short and I'll let you know.

Thanks for reporting

**Comment by luigidellaquila** [comment]
fixed in branches 2.0.x, 2.1.x and develop (2.2)
